src/components/CentralContainer/Sidebar/SidebarTabs/PanelTitleButton.css

```diff
@@ -0,0 +1,5 @@
+.tab-panel-title-button {
+    z-index: var(--ylv-panel-title-button-z-index) !important;
```

Comment:
Why do we need `!important` here? Does JoyUI specify any z-index order on buttons?